### PR TITLE
DUOS-414 - DUOS UI: Update Pagination Method

### PR DESCRIPTION
**Pull request by jaime-bernales-broad**
**[JIRA Ticket](https://broadinstitute.atlassian.net/browse/DUOS-414)**

In the Jira ticket you can find the mockup with the final design.

**Tables to be updated can be found on the following pages:**
DAC Console (x2 tables)
Manage Data Use Limitations
Manage Users
Manage Data Access Requests
Dataset Catalog
Reviewed Cases Record (x2)

### CODE REVIEW

File: src/components/PaginatorBar.js [hunk 1]
```diff
@@ -27,6 +27,23 @@ export const PaginatorBar = hh(class PaginatorBar extends Component {
     }, () => { this.props.changeHandler(this.state.limit); });
   }
 
+  firstItem(currentPage, total) {
+    if (total == 0) {
```
Comment (dags) [inline src/components/PaginatorBar.js]: can be optimized, with  : return value == 0 ? value : expression

File: src/components/PaginatorBar.js [hunk 2]
```diff
@@ -27,6 +27,23 @@ export const PaginatorBar = hh(class PaginatorBar extends Component {
...
+      return (currentPage * this.state.limit) - this.state.limit + 1;
+    }
+  }
+
+
+  lastItem(hasNextPage) {
+    if (hasNextPage) {
+      return this.props.currentPage * this.state.limit;
+    } else {
+      return this.props.total;
```
Comment (dags) [inline src/components/PaginatorBar.js]: similar case  here

File: src/components/PaginatorBar.css [hunk 3]
```diff
@@ -65,22 +84,42 @@
...
+}
+
+.select-wrapper {
+  float: right;
+}
+
+.show-results-wrapper,
+.select-wrapper {
+  margin-top: 7px;
+}
```
Comment (rushtong) [inline src/components/PaginatorBar.css]: nit: add newline

**Review by rushtong — APPROVED** [review]
Looks good 👍, pending Diego's feedback.

**Review by dags — APPROVED** [review]

**Pull request merged**